land_claims.civmap.json

```diff
@@ -241,5 +241,13 @@
+{"id":"ac713034-c3b1-4654-bd22-c9370ec06fa6","collectionId":"civmap:collection/user","polygon":[[[7586,-7109],[7925,-7106],[7931,-6861],[7638,-6887],[7548,-7025],[7581,-7106]]],"name":"Union (Buckeye)"},
```

Comment:
`"collectionId":"civmap:collection/user"` is unnecessary and shouldn't be included. some day I'll fix this on the map itself